src/Contrib.KafkaFlow.Outbox.SqlServer/schema/0001.Shema.sql

```diff
@@ -0,0 +1,5 @@
+IF NOT EXISTS(SELECT * FROM sys.databases WHERE name = 'kafkaflowtest')
+BEGIN
+	CREATE DATABASE [kafkaflowtest]
```

Comment:
I think you should be creating the schema here, not the database. This might work for tests... but wont work for production environments. We cant assume the name of the DB there.

For tests - I'd be looking at using [MSSQL Test Containers](https://dotnet.testcontainers.org/modules/mssql/). @AlexeyRaga would you consider that + DbUp possibly embedded in the code startup (or similar)?

Back to here - I think this should be:
```
IF NOT EXISTS(SELECT * FROM sys.schemas WHERE name = 'outbox')
BEGIN
	CREATE DATABASE [outbox]
END
```

There'll be similar changes below too.

Reply:
I am happy to take the test containers route. 
I have never done it myself though, so can't speak from experience :)